packages/core/src/compiler.ts

```diff
@@ -74,7 +74,8 @@ export default class Compiler {
     if (typeof src === 'object') {
       const result: StringStructure = {};
       for (const key of Object.keys(src)) {
-        const pl = (plurals && plurals[key]) || plural;
+        const normalizedPluralsKey = key.length > 2 ? normalize(key) : key;
```

Comment:
could we just `normalize` 💯 % of the time? seems like this length check is already implemented there. may be nice to dedupe it

Reply:
`normalize` will throw if given unexpected input, so no.